include/zephyr/dt-bindings/pinctrl/renesas/pinctrl-rcar-common.h

```diff
@@ -71,6 +73,14 @@
 #define IP1SR5(shift, func) IPnSR(1, 5, shift, func)
 #define IP2SR5(shift, func) IPnSR(2, 5, shift, func)
 #define IP3SR5(shift, func) IPnSR(3, 5, shift, func)
+#define IP0SR6(shift, func) IPnSR(0, 6, shift, func)
```

Comment:
I believe that this should go into separate patch

Reply:
done